package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-mathmagician",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Thi-Ponce/React-Mathmagician.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Thi-Ponce/React-Mathmagician/issues"
+  },
+  "homepage": "https://github.com/Thi-Ponce/React-Mathmagician#readme",
+  "devDependencies": {
+    "@babel/core": "^7.17.7",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/plugin-syntax-jsx": "^7.16.7",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
- [ ] Your `package.json` does not have scripts to start a server check if you created your react project well with `npx create-react-app` it should look like this
![json](https://user-images.githubusercontent.com/44745855/158680121-99a7356a-5c59-485c-ab69-03d02fb45ae1.png)

Reply:
Hey there! Thanks for the review, I did follow the steps on the documentation so I'm not sure what could've cause this error. But I'll take a look!

Reply:
Hello! The package.json file on the my-app folder is exactly like the one you sent me but if I try to just move them it doesn't work. Do you have any feedback on what I could do?